setup.py

```diff
@@ -25,7 +24,7 @@
 
 setup(
     name="SCAutolib",
-    version="3.1.2",
+    version="3.1.3",
```

Comment:
nit: Since this is kind of a big change, in the sense that from now on Fedora will be supported, can you please change the version to 3.2.0 instead of 3.1.3?

Reply:
changed to 3.2.0.